problems/isOdd.js

```diff
@@ -7,8 +7,12 @@
 
 */
 
-function isOdd() {
-
+const isOdd = (n) => {  
+    if(n % 2 === 1){
```

Comment:
If you use an implicit return with this conditional, do you need an else if statement to return false?